### ISSUE TITLE
Improve documentation and examples for lithostratigraphicTerms

### ISSUE THREAD

**Issue by dennereed** [tdwg/paleo#23]
The current Darwin Core definition and wiki documentation is sparse. Expand documentation and provide examples. Should facies and other geological terms be included here? e.g. "sandstone in upward fining sequence"; "mudstone interpreted as paleolake margin habitat"

**Comment by hollyel** [comment]
Some exploration of related topics that will need to be rolled up into the documentation/examples suggested here:
#40 
#39

**Comment by RogerBurkhalter** [comment]
We need to be very careful here about conflating terms. Base on the DwC examples, the DwC:lithostratigraphicTerms is text string of the terms used to describe the chronostratigraphy, i.e. the example used is Pleistocene-Weichselien, describing the last glaciation in Europe during the Pleistocene. This has nothing to do with lithologic descriptions, i.e. limestone|packstone, siltstone, etc. that describes the "matrix" where the fossil occurs (thus helping define the depositional environment the fossilized animal lived. The question asked by dennereed has everything to do with actual lithologic descriptions, which are not covered by DwC.

In my database, I typically store the lithologic data with the locality, because 1. most of my localities are single horizons from a measured section, and 2. everything collected from that locality has the same lithology. I also record sedimentary structures associated with the single horizon (sole marks, cross-bedding, etc. in the locality information.

Because these "localities" are a part of a measured section, I also record a "parent locality" that is used to both describe the base of the measured section. i.e. "began at the top of the 2-meter thick massive limestone near the road" and I record additional geological terms, including larger-scale sedimentary structures. At this point, I do not include sequence strat or other interpretations. The hope is that soon there will be a Python routine to help with those interpretations. I use MS  SQLServer 2019 with embedded R and Python.

**Comment by RogerBurkhalter** [comment]
As you can probably tell, my database is quite atomized. Most of the data I have is not shared in DwC because the lack of corresponding fields. It is also perhaps more than the needs of a distributed database.

**Comment by hollyel** [comment]
@RogerBurkhalter I totally agree about being careful with the terms. I actually read the title of this issue incorrectly! I see now that it is meant to be "examples for lithostratigraphicTerms" as in the dwc:lithostratigraphicTerms term, not "examples for lithostratigraphic terms" as in all of the related terms in dwc :)

So, amending my previous comment - I do think that the issues relate to one another, but the documentation part doesn't need to be "rolled up" here. There is a big need for better documentation in general for all lithostratigraphic terms in DwC and for expanded capability for supporting lithostratigraphic information that we cannot share well at the moment. I actually suggested this term in one of the other threads because of the lack of clear documentation for it and I wondered if the example given actually makes sense for the intended use of the term since there isn't a good definition. The name and example don't quite match up to me. I'm not suggesting we just start using it differently, but that it's worth exploring how it has been used and what the intended use actually is.

**Comment by tucotuco** [comment]
As I also commented in Issue #39,  would you say the examples are wrong? Or
misleading? Incomplete?  Examples are not normative, so can be changed
without a lot of process. The same is true of comments.

**Comment by hollyel** [comment]
@tucotuco I'm going to move my comment from the other thread here since it is more aligned with this discussion - it will disappear in the other thread shortly. To me it seems like the example is wrong? or maybe just incomplete?

The definition seems to get at how we could use dwc:lithostratigraphicTerms for the string of unit names, not sure about descriptions. However, it seems that the current example for dwc:lithostratigraphicTerms is actually more in reference to chronostratigraphy. Therefore, it would not be a place to put lithostratigraphic units or more descriptive information about the lithology. Can you share any further information about the intended purpose of this term when it was created? And while you're here, how one might represent multiple values or ranges for a member in #39 ? THANK YOU! (also related discussions in #40 )

**Comment by hollyel** [comment]
Disclaimer, stratigraphy not my expertise... documentation and data I can do! :)

**Comment by RogerBurkhalter** [comment]
I would suggest we use formal definitions that have been vetted for these terms. The classic example is the IUGS International Stratigraphic Guide, A Guide to Stratigraphic Classification, Terminology, and Procedure (2nd Edition, 1994) by Amos Salvador, published by the IUGS and GSA. Although now somewhat old, it is still a great guide with pre-made definitions and the correct usage of those terms. Some of the content is in Google Books:
https://www.google.com/books/edition/_/l03YnbnoxzUC?hl=en&gbpv=1&pg=PA15&dq=salvador+1994+stratigraphy
 Newer editions or more recent guides exist. Anyone?
